docs/source/api/python/dgl.sparse_v0.rst

```diff
@@ -171,10 +171,7 @@ Matrix Multiplication
 .. autosummary::
     :toctree: ../../generated/
 
-    spmm
-    bspmm
-    spspmm
-    mm
+    matmul
     sddmm
     bsddmm
```

Comment:
I remember our decision is still keeping `spmm`, `bspmm` and `spspmm` but will not mention them in the colab tutorial.

Reply:
I thought we keep the interfaces of these ops but remove them from the doc. Let's have a discussion on it today.